### PR TITLE
chore(release): v10.37.1

### LINKED ISSUE #8924: v10.37.1
**Status:** Stable
**Preview date:** TODO
**Target date:** TODO
**Changelog:**  

# [v10.37.1](https://github.com/carbon-design-system/carbon/compare/v10.37.0...v10.37.1) (2021-6-18)
### Bug fixes :bug:
- fix(Search): restore default component size (#8912) ([`d06886bab`](https://github.com/carbon-design-system/carbon/commit/d06886bab))

### Housekeeping :house:
- chore(release): v10.37.1-rc.0 ([`53f5f5b1b`](https://github.com/carbon-design-system/carbon/commit/53f5f5b1b))

### Timeline

# Fill out after publishing a version
- Wednesday, TODO

### Release checklist

- [ ] Determine scope of changed items (and dependencies)
- [ ] Verify new behavior
- [ ] Verify no regressions have occurred
- [ ] Create release candidate and PR against internal properties
- [ ] Create stable release and publish to next
- [ ] Push git tag and create changelog
- [ ] Switch next tag to latest for changed packages

**Browser support**

- [ ] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Edge
- [ ] IE11

**Screen reader support**

- [ ] JAWS
- [ ] NVDA
- [ ] VoiceOver on macOS
- [ ] VoiceOver on iOS

### Release radar participants

- [ ] Carbon Website
- [ ] [IBM Cloud](https://ibm.biz/Bdq2qr)
- [ ] [IBM Dotcom](https://github.com/carbon-design-system/ibm-dotcom-library)
  - set the label to "package: react" for deploy previews
- [ ] [IBM Security](https://github.com/carbon-design-system/ibm-security)
- [ ] [IBM Watson IoT](http://github.com/ibm/carbon-addons-iot-react)
- [ ] [Carbon Angular](https://github.com/ibm/carbon-components-angular)

### PR DESCRIPTION
**Pull request by jnm2377**
Closes #8924

patch release for `v10.37.1-rc.0`

### CODE REVIEW

File: packages/react/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "carbon-components-react",
   "description": "The Carbon Design System is IBM’s open-source design system for products and experiences.",
-  "version": "7.37.0",
```
Comment (joshblack) [inline packages/react/package.json:4]: Seems like the version bumps are patches but the title is `rc.0`? Wasn't sure which one it was 👀

Comment (jnm2377) [reply packages/react/package.json:4]: @joshblack I was a bit confused by the commit instructions on that
Screen Shot 2021-06-18 at 2 59 17 PM

Comment (jnm2377) [reply packages/react/package.json:4]: I suppose it's just a patch. 👀

**Review by joshblack — APPROVED** [review]

**Comment by jnm2377** [comment]
@joshblack should I close this PR after creating the release?

**Pull request closed without merging**